src/utilities/TimeUtilities.ts

```diff
@@ -190,4 +190,67 @@ export namespace TimeUtilities {
 
         return finalDate;
     }
+
+    /**
+     * Options for making a Discord timestamp from the bot's side
+     * @typedef {Object} DiscordTimestampOptions
+     * @property {Date | number} time The time to show
+     * @property {TimestampType} [style] The style to show
+     */
+
+    /** 
+     * Creates a Discord timestamp
+     * @param {DiscordTimestampOptions} [options={}] Options to create the timestamp
+     * @returns {string} The Discord timestamp
+     */
+    export function getDiscordTime({ time = Date.now(), style = TimestampType.Relative } = {}): string {
+        // Truncate and divide by 1000 to transform  ms to seconds
+        return `<t:${Math.trunc(time / 1000)}:${style}>`; 
+    }
+
+    export enum TimestampType {
```

Comment:
You can just put the enum outside of the namespace. That way, you don't end up with a super long `TimeUtilities.TimestampType.Something` in every line that calls this method.